VimeoNetworking/VimeoNetworking/Models/Spatial.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Spatial.swift
+//  Pods
+//
+//  Created by Lim, Jennifer on 1/6/17.
+//
+//
+//
+
+public class Spatial: VIMModelObject
+{
+    enum Projection: String
+    {
+        case Equirectangular = "equirectangular"
+        case Cylindrical = "cylindrical"
+        case Cubical = "cubical"
+        case Pyramid = "pyramid"
+        case Dome = "dome"
+    }
+    
+    enum Format: String
+    {
+        case Mono = "mono"
+        case LeftRight = "left-right"
+        case TopBottom = "top-bottom"
+    }
+    
+    // MARK: - Properties
+    
+    // TODO: - Update to be Projection, Format types
+    public dynamic var projection: String?
+    public dynamic var stereoFormat: String?
```

Comment:
I wonder if it's worth making `typealias`es for these strings. So that the Format enum is also of type `FormatType`, which would be a typealias for String. And then something similar for projection.